libs/community/langchain_community/tools/sql_database/tool.py

```diff
@@ -45,8 +45,17 @@ def _run(
         query: str,
         run_manager: Optional[CallbackManagerForToolRun] = None,
     ) -> Union[str, Sequence[Dict[str, Any]], Result]:
+        # add include_columns as valid args for db.run_no_throw
+        valid_kwargs = {}
+        if self.metadata:
```

Comment:
metadata isn't meant to affect tool outputs. include_columns should be its own tool on QuerySQLDataBaseTool